### PR TITLE
chore: tune parcoords [DET-4991]

### PR DESCRIPTION
**Pull request by hkang1**
## Description

Tasks
- add filtered table to PC plot
- show all trials and then filter based off window
- make column order reset to default order (unable to lock or prevent the columns from being re-ordered)
- fiddle with the labels to make them stand out but without taking even more space (angle them to 45)
- truncate the hp label to 20 characters for long labels
- limit the hp to 10
- add tips on how to filter on parallel coordinates

Thought process on color scale gradients
- avoid red/green due to it being the most common color blindness
- use blue/red color scale for searcher metric, for when we do know if a metric is good or bad within a range. The color gradient is reversed when `smallerIsBetter` is true.
- use blue/orange color scale for non-searcher metric to avoid using red, which generally suggests something is bad. orange for neutral color that sharply contrasts against blue being on the opposite side of the color wheel from blue

chart with searcher metric selected
Screen Shot 2021-02-16 at 4 46 34 PM

chart with non-searcher metric selected
Screen Shot 2021-02-16 at 5 16 15 PM

filtered applied
Screen Shot 2021-02-16 at 5 22 45 PM

## Checklist

- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](../docs/release-notes/README.md) for details.

[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234

### CODE REVIEW

File: webui/react/src/utils/color.test.ts [hunk 1]
```diff
@@ -0,0 +1,129 @@
+import { GLASBEY } from 'constants/colors';
```
Comment (hamidzr) [inline webui/react/src/utils/color.test.ts:1]: 👍🏼

File: webui/react/src/utils/color.ts [hunk 2]
```diff
@@ -1,35 +1,61 @@
...
   s: number;
 }
 
+/*
+ * r - red between 0 and 255
+ * g - green between 0 and 255
+ * b - blue between 0 and 255
+ * a - alpha between 0.0 and 1.0
+ */
+interface RgbaColor {
```
Comment (hamidzr) [inline webui/react/src/utils/color.ts:20]: are there libraries we could use that do these color conversions for us?

**Review by hamidzr — COMMENTED** [review]
other than some issues with setting and moving hp filter ranges this looks good to me. while I edit the range and drag it up and down sometimes there range indicator jumps backward

File: webui/react/src/utils/color.ts (hunk 2 above)
Comment (hkang1) [reply webui/react/src/utils/color.ts:20]: this one looks promising but it adds a decent amount in size (7.6kB) for something so small...

https://www.npmjs.com/package/color
https://bundlephobia.com/result?p=color@3.1.3

Comment (hamidzr) [reply webui/react/src/utils/color.ts:20]: hmm since you've added a test for it as well using our own implementation sgtm

**Review by hamidzr — APPROVED** [review]
LGTM. We talked about the issue of the hp filter shifting under the cursor goes back to rerendering while the user is moving the filter and it can be helped by eliminating/reducing rerenders (in another pr)

**Pull request merged**